assembl/static2/js/app/components/debate/brightMirror/sideComment/innerBoxView.jsx

```diff
@@ -0,0 +1,74 @@
+// @flow
+import React from 'react';
+
+import moment from 'moment';
+import activeHtml from 'react-active-html';
+import { I18n } from 'react-redux-i18n';
+
+import AvatarImage from '../../../common/avatarImage';
+import { transformLinksInHtml /* getUrls */ } from '../../../../utils/linkify';
+import { postBodyReplacementComponents } from '../../common/post/postBody';
+
+export type Props = {
+  contentLocale: string,
+  extractIndex: number,
+  extracts: ?Array<FictionExtractFragment>,
+  comment: ExtractComment,
+  changeCurrentExtract: ?Function
```

Comment:
same as the other, Functions which arent't 'refetch' and things like that should be described ideally :)